packages/compiler-cli/src/ngtsc/typecheck/src/type_check_block.ts

```diff
@@ -1253,6 +1279,32 @@ class TcbSwitchOp extends TcbOp {
   }
 }
 
+/**
+ * A `TcbOp` which renders a `for` block as a TypeScript `for...of` loop.
+ *
+ * Executing this operation returns nothing.
+ */
+class TcbForOfOp extends TcbOp {
+  constructor(private tcb: Context, private scope: Scope, private block: TmplAstForLoopBlock) {
+    super();
+  }
+
+  override get optional() {
+    return false;
+  }
+
+  override execute(): ts.Expression|null {
```

Comment:
```suggestion
  override execute(): null {
```
nit: Can we change this to just `null`? It almost reads like a bug that the return type explicitly mentions a `ts.Expression` might be returned but the function always just returns `null`.